22/homework_22.6

```diff
@@ -0,0 +1,5 @@
+
+SELECT COUNT(purchase.id) as total_purchases, books.author
```

Comment:
The result should contain a table with only one column, `amount`. You have 2 columns instead